lib/Models/ArcGisFeatureServerCatalogGroup.js

```diff
@@ -0,0 +1,213 @@
+var loadJson = require('terriajs-cesium/Source/Core/loadJson');
+var when = require('terriajs-cesium/Source/ThirdParty/when');
+
+var TerriaError = require('../Core/TerriaError');
+var CatalogGroup = require('./CatalogGroup');
+var inherit = require('../Core/inherit');
+var proxyCatalogItemUrl = require('./proxyCatalogItemUrl');
+var replaceUnderscores = require('../Core/replaceUnderscores');
+var ArcGisFeatureServerCatalogItem = require('./ArcGisFeatureServerCatalogItem');
+
+/**
+ * A {@link CatalogGroup} representing a collection of layers from an ArcGIS Feature Service.
+ * Eg. https://sampleserver6.arcgisonline.com/arcgis/rest/services/Wildfire/FeatureServer
+ *
+ * @alias ArcGisFeatureServerCatalogGroup
+ * @constructor
+ * @extends CatalogGroup
+ *
+ * @param {Terria} terria The Terria instance.
+ */
+var ArcGisFeatureServerCatalogGroup = function(terria) {
+    CatalogGroup.call(this, terria, 'esri-featureServer-group');
+
+    /**
+     * Gets or sets the URL of the Feature Server.  This property is observable.
+     * @type {String}
+     */
+    this.url = '';
+
+    /**
+     * Gets or sets a description of the custodian of the data sources in this group.
+     * This property is an HTML string that must be sanitized before display to the user.
+     * This property is observable.
+     * @type {String}
+     */
+    this.dataCustodian = undefined;
+
+    /**
+     * Gets or sets a hash of names of blacklisted data layers.  A layer that appears in this hash
+     * will not be shown to the user.  In this hash, the keys should be the Title of the layers to blacklist,
+     * and the values should be "true".  This property is observable.
+     * @type {Object}
+     */
+    this.blacklist = undefined;
+
+    /**
+     * Gets or sets a hash of properties that will be set on each child item.
+     * For example, { 'treat404AsError': false }
+     */
+    this.itemProperties = undefined;
+
+    knockout.track(this, ['url', 'dataCustodian', 'blacklist', 'itemProperties']);
+};
+
+inherit(CatalogGroup, ArcGisFeatureServerCatalogGroup);
+
+defineProperties(ArcGisFeatureServerCatalogGroup.prototype, {
+    /**
+     * Gets the type of data member represented by this instance.
+     * @memberOf ArcGisFeatureServerCatalogGroup.prototype
+     * @type {String}
+     */
+    type : {
+        get : function() {
+            return 'esri-featureServer-group';
+        }
+    },
+
+    /**
+     * Gets a human-readable name for this type of data source, such as 'Web Map Service (WMS)'.
+     * @memberOf ArcGisFeatureServerCatalogGroup.prototype
+     * @type {String}
+     */
+    typeName : {
+        get : function() {
+            return 'ArcGIS Feature Server Group';
+        }
+    },
+
+    /**
+     * Gets the set of functions used to serialize individual properties in {@link CatalogMember#serializeToJson}.
+     * When a property name on the model matches the name of a property in the serializers object lieral,
```

Comment:
ha! I copied that directly from another CatalogItem. Looks like I'm not the only one - try searching for "lieral" in the /lib directory, there are 23 hits :-)